### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.8 es 4.35 (6 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#313] [Esercizi M-Z]
![es.35](https://github.com/user-attachments/assets/08806685-670d-4918-a955-519a644b91e9)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/0e1ee405-e82c-4e05-bc41-b0a96011c8f6)

**Comment by rimaout** [comment]
<img width="912" alt="image" src="https://github.com/user-attachments/assets/ca15a6b3-455b-436a-aeed-a26605291869" />
